Update UG documentation (#375)
AY2223S1-CS2103T-T12-1/tp: docs/index.md

![Ui](images/Ui.png)

> **TA-Assist is a desktop application for managing your students' grades.** While it has a GUI (Graphical User Interface), most of the user interactions happen from its CLI (Command Line Interface).

Reviewer: ```suggestion
**TA-Assist is a desktop application for managing your students' grades.** While it has a GUI (Graphical User Interface), most of the user interactions are performed through a CLI (Command Line Interface).
```

Author: Alright sure!
